### PR TITLE
proposed fix for #574

### PR DESCRIPTION
**Pull request by heapifyman**
I don't think I got all the edge cases but at least tinymce should not crash anymore when encountering an unsupported locale / language.

### CODE REVIEW

File: tinymce4-parent/tinymce4/src/main/java/wicket/contrib/tinymce4/settings/TinyMCESettings.java [hunk 1]
```diff
@@ -415,10 +411,138 @@ private Mode(String name)
...
+        private static final Map<String, Language> unmappedLocales = new HashMap<>();
+
+        static {
+        	unmappedLocales.put("bg", Language.bg_BG);
+        	unmappedLocales.put("en", Language.en_GB);
+			unmappedLocales.put("fr", Language.fr_FR);
+			unmappedLocales.put("hi", Language.hi_IN);
+			unmappedLocales.put("hu", Language.hu_HU);
+			unmappedLocales.put("is", Language.is_IS);
+			unmappedLocales.put("iw", Language.he_IL);
```
Comment (martin-g) [inline tinymce4-parent/tinymce4/src/main/java/wicket/contrib/tinymce4/settings/TinyMCESettings.java]: The indentation is off.

File: tinymce4-parent/tinymce4/src/main/java/wicket/contrib/tinymce4/settings/TinyMCESettings.java [hunk 2]
```diff
@@ -415,10 +411,138 @@ private Mode(String name)
...
+            }
+            language = fromString(locale.getLanguage());
+            if (language != null) {
+                return language;
+            }
+            language = fromString(locale.getISO3Language());
+            if (language != null) {
+            	return language;
+			}
+			return unmappedLocales.get(locale.getLanguage());
```
Comment (martin-g) [inline tinymce4-parent/tinymce4/src/main/java/wicket/contrib/tinymce4/settings/TinyMCESettings.java]: Indentation issues

File: tinymce4-parent/tinymce4/src/main/java/wicket/contrib/tinymce4/settings/TinyMCESettings.java (hunk 1 above)
Comment (heapifyman) [inline tinymce4-parent/tinymce4/src/main/java/wicket/contrib/tinymce4/settings/TinyMCESettings.java]: I've applied [wicket style settings](https://cwiki.apache.org/confluence/display/WICKET/Wicket+Source+Code+Style).
Hope that fixes it.

File: tinymce4-parent/tinymce4/src/main/java/wicket/contrib/tinymce4/settings/TinyMCESettings.java (hunk 2 above)
Comment (heapifyman) [inline tinymce4-parent/tinymce4/src/main/java/wicket/contrib/tinymce4/settings/TinyMCESettings.java]: I've applied [wicket style settings](https://cwiki.apache.org/confluence/display/WICKET/Wicket+Source+Code+Style).
Hope that fixes it.

**Comment by tahitiangabriel** [comment]
Hi,
I'm having the same issue for "fr_FR" local with wicket 7.x.
Is it also possible to cherry pick this fix into the 7.x branch?

**Pull request merged**